README.md

```diff
@@ -81,17 +81,23 @@ vars:
     event_properties_to_pivot: ['the', 'list', 'of', 'property', 'fields'] # Note: this is case-SENSITIVE and must match the casing of the property as it appears in the JSON
 ```
 
-And if there are columns in your source `mixpanel.event` table that are not the Mixpanel default columns, add the following variable to your `dbt_project.yml` file to include them:
+### Passthrough Columns
+
+Additionally, this package includes all standard source `EVENT` columns defined in the `staging_columns` macro. You can add more columns using our passthrough column variables. These variables allow the passthrough fields to be aliased (`alias`) and casted (`transform_sql`) if desired, although it is not required. Datatype casting is configured via a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - this will be dealt with by the alias attribute - and your custom passthrough fields will be casted accordingly.
+
+Use the following format for declaring the respective passthrough variables:
 
 ```yml
 # dbt_project.yml
 
-...
-config-version: 2
-
 vars:
   mixpanel:
-    event_custom_columns: ['the', 'list', 'of', 'column', 'names']
+    event_custom_columns:
+      - name:           "property_field_id"
+        alias:          "new_name_for_this_field_id"
+        transform_sql:  "cast(property_field_id as int64)"
```

Comment:
the staging event model is configured slightly differently from how we usually make our staging tables. the column aliasing happens in the macro, and there's one less CTE. this means that the `transform_sql` snippet needs to reference the original name of the passthrough column here (instead of the new field name, which is the case in our other packages). the alternative would be to reconfigure the staging model to match up with how we usually do it; it would just take a little time to move all of the aliasing from the macro to the staging model

i lean toward leaving it as is, but i see the value of consistency across packages and am very open to opinions 👂